Dockerfile

```diff
@@ -28,13 +23,17 @@ ENV C_INCLUDE_PATH=/usr/include/gdal
 RUN apt-get install -y binutils libproj-dev gdal-bin libjpeg-dev
 
 # Run the application:
-COPY disasterinfosite /disasterinfosite
```

Comment:
It's likely that you'll still need `disasterinfosite` on these images because a bunch of supporting files (like the shapefile data) are in there.